q2cli/builtin/tools.py

```diff
@@ -80,40 +80,149 @@ def export_data(input_path, output_path, output_format):
     click.echo(CONFIG.cfg_style('success', success))
 
 
-def show_importable_types(ctx, param, value):
-    if not value or ctx.resilient_parsing:
-        return
+def print_descriptions(descriptions, tsv):
+    if tsv:
+        for value, description in descriptions.items():
+            click.echo(f"{value}\t", nl=False)
+            if description:
+                click.echo(deformat_description(description))
+            else:
+                click.echo()
+    else:
+        import textwrap
+        for value, description in descriptions.items():
+            click.secho(value, bold=True)
+            if description:
+                description = deformat_description(description)
+                tabsize = 8
+                wrapped_description = textwrap.wrap(description,
+                                                    width=72-tabsize,
+                                                    initial_indent='\t',
+                                                    subsequent_indent='\t',
+                                                    tabsize=tabsize)
+                for line in wrapped_description:
+                    click.echo(f"{line}")
+            else:
+                click.secho("\tNo description", italic=True)
+            click.echo()
+
+
+def deformat_description(description):
+    import re
+    deformatted = re.sub(r"[\t\n]+", ' ', description)
+    despaced = re.sub(r" +", ' ', deformatted)
+    return despaced
+
+
+def get_matches(words, possibilities, cutoff=0.5):
```

Comment:
How'd you come up with 0.5 as the default? It feels a little low to me, and on a few tests it seems pretty permissive, e.g.:

```bash
$ qiime tools show-types mpledata
ImmutableMetadata
	Immutable sample or feature metadata.

SampleData[AlphaDiversity]
	Alpha diversity values, each associated with a single sample
	identifier.
...
```

The match to `ImmutableMetadata` seems pretty distant. 

I don't have a good feel for what it should be, so mostly I'm wondering if you had a specific motivation for 0.5.

Reply:
No specific motivation, having results that a user is not interested in seemed preferable to omitting results a user is interested in. I'll change to 0.6.